### DISCUSSION TITLE
[blocks]: Kanban Board with Todo List view

### DISCUSSION THREAD

**Discussion by chirag-mehta** [shadcn-ui/ui#3171] [Blocks Request]
### Description

It's a kanban board. Can use [React DnDKit Sortables](https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-vertical--basic-setup)

Three default columns where you can drag columns and drag cards. Linear like list view with the same data would be super helpful imo for productivity applications and building this from scratch is always a pain.

### Example

https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-vertical--basic-setup

**Comment by chaha0s** [comment]
This would be good to have!

**Comment by ckluis** [comment]
Came here to add this request myself.

**Comment by DeoEsor** [comment]
+1

**Comment by n3fquick** [comment]
Any update on this?

**Comment by HeyClos** [comment]
This would be super helpful. Any update on this?

**Comment by Steveb599** [comment]
Any update on this?
